src/pages/en/persona.md

```diff
@@ -0,0 +1,603 @@
+
+In our personas, we included a variety of disabilities and a spectrum of limitations:
+• Permanent disabilities
+• Temporary limitations
+• Situational disabilities/limitations
+
+We believe it is important to account for a wide range of human experiences. This is why for each disability type we included different possibilities of exclusion.
+
+### Seven (7) Persona
+
+5 broad disability types and 2 combined disability types
+
+- [Brandon – Auditory](#Brandon)
+- [Alasie – Vision](#Alasie)
+- [Yenah – Dexterity](#Yenah)
+- [Ike – Mobility](#Ike)
+- [Damien – Cognitive](#Damien)
+- [Valeria – Cognitive and Auditory](#Valeria)
+- [Yosef – Mobility and Speech](#Yosef)
+
+<div class="container">
+
+### Brandon – Auditory {#Brandon}
+
+<div class="row">
+
+<div class="col-md-6">
+
+#### <i class="far fa-address-card" aria-hidden="true" ></i> About
+
+<dl>
+  <dt>Pronouns</dt>
+  <dd> He/Him</dd>
+  <dt>Age</dt>
+  <dd> 52</dd>
+  <dt>Job</dt>
+  <dd> User Experience (UX).</dd>
+  <dt>Commonly used sites</dt>
+  <dd> IAAP, CSPS.</dd>
+  <dt>Disability:</dt>
+  <dd>Deaf since birth.
+  <p>Example of Temporary or situational</p>
+  <ul>
+    <li>Temporary hearing loss due to an ear infection in the right ear.</li>
+    <li>Construction noise.</li>
+  </ul>
+  </dd>
+</dl>
+</div>
+</div>
+<div class="row">
+<div class="col-md-6">
+
+#### <i class="fas fa-signal" aria-hidden="true"></i> Skills (aptitude)
+
+- Tech-savvy and able to picks up new software quickly.
+- Knows sign-language.
+</div>
+<div class="col-md-6">
+
+#### <i class="fas fa-tools" aria-hidden="true"></i> Tools (adaptive technology, devices, other)
+
+Category: audio replacement and enhancement.
+
+- Live closed captioning in meetings.
+- Other: Transcripts during/post-event.
+</div>
+</div>
+
+<div class="row">
+<div class="col-md-6">
+
+#### <i class="fas fa-notes-medical" aria-hidden="true"></i> Needs (tasks, environments)
+
+- Prefers the option to work from home.
+- Requires access to his laptop daily for work.
+- Often asks for instructions or conversations to be repeated to him.
+</div>
+<div class="col-md-6">
+
+#### <i class="fas fa-hand-paper" aria-hidden="true"></i> Barriers & frustrations (specifics, attitudes)
+
+- It can be really frustrating when tools lag or aren't available.
+- Web-based services and applications that rely on interactions using voice-only commands are really difficult or sometimes not possible to use.
+- Group meetings, back-to-back meetings, or just too many meetings in a day can be draining.
+
+</div>
+</div>
+
+<div class="row">
+<div class="col-md-12">
+
+> #### <i class="fas fa-quote-left" aria-hidden="true"></i> Quote:
+>
+> “When music is playing while a speaker is speaking, it’s very distracting. WCAG 2.1 has a criterion that says background noise has to be certain decibels below the speaker’s voice but it’s an AAA level. This means it’s not mandatory for companies and organizations to comply with this criterion.”
+
+</div>
+</div>
+</div>
+
```

Comment:
Rinse and repeat 😄 

Do these changes in all the sections, basic HTML should be:

`div.row>div.col-md-6*5+div.col-md-12` type this out in your VS Code and after you type the 2, hit tab and watch the magic happen. You also need to make sure the file type is set to HTML. 

VS Code has [Emmet](https://emmet.io) build in.

``` html
<div class="row">
	<div class="col-md-6"></div>
	<div class="col-md-6"></div>
	<div class="col-md-6"></div>
	<div class="col-md-6"></div>
	<div class="col-md-6"></div>
	<div class="col-md-12"></div>
</div>
```